### DISCUSSION TITLE
Technical Gest Author enquiry

### DISCUSSION THREAD

**Discussion by lebe24** [codemagic-ci-cd/codemagic-docs#2257] [Q&A]
Hello code magic Team,

Hope this message finds you well, I would like to inquire about your Gest Author program is still running and like to get a feedback on the article I proposed to your blog

Thank you

**Comment by icarusdust** [comment]
Hi @lebe24, please let me know what article exactly you are referring to.

**Comment by lebe24** [comment]
hi @icarusdust I proposed an article to the Code Magic blog on.

**Elevate Your Flutter Application: Mastering Dynamic Animation with the Flutter Animate Library.**

**Overview:** 
The article will serve as a valuable resource for Flutter developers looking to enhance their app's user experience through animation and the significance of dynamic animations in modern app design and introduce the Flutter Animate library as the primary tool for crafting these animations.

I would be glad to get some feedback on the proposed article

**Reply by icarusdust** [reply]
Thanks for elaborating on it. We will let you know in case of any updates on your suggestion.
